sdk/storage/AzureStorageBlob/Source/Operation/BlobDownloadOperations.swift

```diff
@@ -96,10 +96,16 @@ internal class BlobDownloadInitialOperation: TransferOperation {
 
         let group = DispatchGroup()
         group.enter()
-        downloader.initialRequest { result, _ in
+        downloader.initialRequest { result, httpResponse in
             switch result {
             case .success:
+                guard let responseHeaders = httpResponse?.headers else {
+                    assertionFailure("Response headers not found.")
+                    return
+                }
+                let blobProperties = BlobProperties(from: responseHeaders)
                 parent.totalBytesToTransfer = Int64(downloader.totalSize)
+                parent.bytesTransferred += Int64(blobProperties.contentLength ?? 0)
```

Comment:
Would result in the initial block not being "counted", resulting in "completed" blobs only going to 90+%.